appmonitoring/scripts/validate-aks-charts.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
Will a random failure in these bash files cause the pipeline to fail? Let's ensure that's the case.

Reply:
this is very deterministic code, this should not fail. if it does,it would indicate a code flaw